pkg/kernel/const_linux.go

```diff
@@ -14,6 +16,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:build linux
 // +build linux
```

Comment:
Do we still need this tag since we have `go:build linux`?
```suggestion
```

Reply:
Yes, I think we have to add it. Otherwise linter prints `pkg/kernel/const_linux.go:18: File is not gofmt-ed with -s (gofmt)`

Reply:
Yes, it's not needed anymore

Reply:
Ok, tests don't work without it. It looks like we have to keep these lines